### PR TITLE
cqlsh: add cql_auth support for rest cqlsh commands

### PR DESCRIPTION
**Pull request by amoskong**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- ~[ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~
- ~[ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~
- ~[ ] All new and existing unit tests passed (`hydra unit-tests`)~
- ~[ ] I have updated the Readme/doc folder accordingly (if needed)~

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ def alter_table_to_in_memory(self, key_space_name="keyspace1", table_name="stand
         compaction_strategy = "%s" % {"class": "InMemoryCompactionStrategy"}
         cql_cmd = "ALTER table {key_space_name}.{table_name} " \
                   "WITH in_memory=true AND compaction={compaction_strategy}".format(**locals())
-        node.remoter.run('cqlsh -e "{}" {}'.format(cql_cmd, node.private_ip_address), verbose=True)
+        self.db_cluster.run_cqlsh(node=node, cql_cmd=cql_cmd)
```
Comment (fruch) [inline sdcm/tester.py]: is that exist in branch-2019.1 ? i.e. run_cqlsh ?

**Comment by amoskong** [comment]
On Mon, Jul 15, 2019 at 6:57 PM Israel Fruchter <notifications@github.com>
wrote:

> *@fruch* commented on this pull request.
> ------------------------------
>
> In sdcm/tester.py
> <https://github.com/scylladb/scylla-cluster-tests/pull/1157#discussion_r303380374>
> :
>
> > @@ -1276,7 +1276,7 @@ def alter_table_to_in_memory(self, key_space_name="keyspace1", table_name="stand
>          compaction_strategy = "%s" % {"class": "InMemoryCompactionStrategy"}
>          cql_cmd = "ALTER table {key_space_name}.{table_name} " \
>                    "WITH in_memory=true AND compaction={compaction_strategy}".format(**locals())
> -        node.remoter.run('cqlsh -e "{}" {}'.format(cql_cmd, node.private_ip_address), verbose=True)
> +        self.db_cluster.run_cqlsh(node=node, cql_cmd=cql_cmd)
>
> is that exist in branch-2019.1 ? i.e. run_cqlsh ?
>

Yes.
https://github.com/scylladb/scylla-cluster-tests/blob/branch-2019.1/sdcm/cluster.py#L2203

It's already used in sdcm/tester.py.

**Comment by amoskong** [comment]
BTW, There is some difference of result process between
db_cluster.run_cqlsh() and node.remoter.run(), so I didn't switch the cqlsh
executions to db_cluster.cqlsh()

File: sdcm/tester.py (hunk 1 above)
Comment (amoskong) [reply sdcm/tester.py]: Yes

**Pull request merged**